qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -176,7 +189,7 @@
             variants:
                 - @default:
                 - with_multi_fs_sources:
-                    no socket_group
+                    only with_cache
```

Comment:
Why change it?

Reply:
Because for with_multi_fs_sources, run 'with_cache' is enough. There is no need to run 'socket_group' and 'inode_file_handles'

- with_cache:
- socket_group:
-  inode_file_handles: